lib/generators/mtl/templates/_variables.scss

```diff
@@ -108,6 +108,10 @@ $chip-bg-color: #e4e4e4 !default;
 $chip-border-color: #9e9e9e !default;
 $chip-selected-color: #26a69a !default;
 $chip-margin: 5px !default;
+$chip-height: 22px !default;
+$chip-line-height: 23px !default;
+$chip-font-size: .7rem !default;
+$chip-font-weight: normal !default;
```

Comment:
1) This is not the default values of Materialize https://github.com/Dogfalo/materialize/blob/master/sass/components/_chips.scss
2) While checking the values, found that those values might be reported here: https://github.com/Dogfalo/materialize/blob/master/sass/components/_chips.scss#L14-L28 So, can you please update accordingly please?